File: backend/MyGarden/DAL/EF/DbModels/Issue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MyGarden.DAL.EF.DbModels
{
    public class Issue
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Img_url { get; set; }
        public bool Is_open { get; set; }

        public ApplicationUser Author { get; set; }
        public int AuthorId { get; set; }
        public Plant Plant { get; set; }
        public ICollection<Answer> Answers { get; set; }
    }
}

File: frontend/src/Models/Issue/Issue.ts
import Plant from "../Plant/Plant";
import User from "../User/User";
import Answer from "./Answer";

class Issue {
    id: number = 0;
    title: string = "";
    author: User = new User();
    plant: Plant = new Plant();
    description: string = "";
    image_path: string = "";
    answers: Answer[] = [];
    is_open: boolean = true;
}

export default Issue;